BUILD

```
@@ -0,0 +1,122 @@
+cc_library(
+    name = "api",
+    srcs = glob(["src/api/*.cc"]),
+    hdrs = glob([
+        "include/maliput/api/*.h",
+        "include/maliput/api/rules/*.h",
+
+        "include/maliput/base/*.h",
+        "include/maliput/common/*.h",
+        "include/maliput/math/*.h",
+    ]),
+    strip_include_prefix = "include",
+)
+
+cc_library(
+    name = "base",
+    srcs = glob(["src/base/*.cc"]),
+    hdrs = glob([
+        "include/maliput/base/*.h",
+    ]),
+    strip_include_prefix = "include",
+    deps = ["@yaml-cpp//:yaml-cpp", "@//:common", '@//:math', '@//:api']
```

@erasmomontes nit: consider using double quotes consistently

```suggestion
    deps = ["@yaml-cpp//:yaml-cpp", "@//:common", "@//:math", "@//:api"],
```

Same elsewhere.